### PR TITLE
Refactor hooks according to the cxflow 1.0 naming convention

### PR DESCRIPTION
**Pull request by blazekadam**
- Refactor hooks according to the cxflow 1.0 naming convention
- Use slash instead of underscore in WriteTensorboard hook

### CODE REVIEW

**Comment by blazekadam** [comment]
Please squeeze

File: cxflow_tensorflow/hooks/write_tensorboard_hook.py [hunk 1]
```diff
@@ -10,19 +10,19 @@
...
     -------------------------------------------------------
     hooks:
-      - class: TensorBoardHook
+      - cxflow_tensorflow.WriteTensorboard
     -------------------------------------------------------
     # unknown variable types are casted to strings
     hooks:
-      - class: TensorBoardHook
+      - cxflow_tensorflow.WriteTensorboard:
         on_unknown_type: str
```
Comment (FloopCZ) [inline cxflow_tensorflow/hooks/write_tensorboard_hook.py]: I believe this should be indented by two more spaces

File: cxflow_tensorflow/model.py [hunk 2]
```diff
@@ -149,7 +149,7 @@ class BaseModel(AbstractModel, metaclass=ABCMeta):   # pylint: disable=too-many-
     """
 
     TRAIN_OP_NAME = 'train_op'
-    LOSS_NAME = 'train_op'
+    LOSS_NAME = 'loss'
```
Comment (FloopCZ) [inline cxflow_tensorflow/model.py]: again, this should be on the top of the file

Comment (FloopCZ) [inline cxflow_tensorflow/model.py]: ok, this is fine

**Comment by blazekadam** [comment]
You may hit the `Update` button right here in GitHub. I ll do it for you this time :) @FloopCZ

**Comment by FloopCZ** [comment]
Pfff, buttons...

**Pull request merged**